### DISCUSSION TITLE
mobile view shows gutter on right side of content

### DISCUSSION THREAD

**Discussion by noyloza** [bootscore/bootscore#285] [Q&A]
I'm trying to figure out why in mobile I'm getting a gutter in the right side of my content for my interior pages.

link: http://laurenb18.sg-host.com/about/

All my pages have this gutter and I can't figure out why. I'm trying to get the content to spread to the entire width of the screen, without that little 10px white gutter. Any ideas?

**Comment by crftwrk** [comment]
It is something in your footer widgets. They have non-responsive inline styles as well. Copy widgets content to an editor and then remove all footer widgets. Your site works now fine in mobile view. Now add widgets step-by-step again and check where it's happen.  

Does this help?

**Comment by noyloza** [comment]
I tried removing all footer content and the problem persisted. I'm unsure what it could be because it's recent and I haven't done anything to the universal parts of the pages.  Should all my content be inside a "container-fluid"? I don't understand why some content does go full screen and other content doesnt. Any other ideas?

**Comment by crftwrk** [comment]
Then problem is in `footer.php`. Please delete file (of course, keep a copy) in your child and try to use the default `footer.php` from main-theme. Then do the same with the widgets again. There are still non-responsive inline-styles in your widgets as well, you should do them in an extra stylesheet with responsive media queries. If I remove all footer content, then your site is fine. Bug is somewhere in the footer.

![screenshot](https://user-images.githubusercontent.com/51531217/196249895-f2b64ada-7acd-4dc9-837a-2df62575916c.png)

![screenshot-1](https://user-images.githubusercontent.com/51531217/196249922-049ab432-5525-4fa9-a8e1-b8c7dd84985f.png)
